### PR TITLE
Update to logo

### PR DESCRIPTION
**Pull request by ehsteve**
A large update to the logo repo which includes
- using open source font
- clean up of original svg files
- addition of script to automatically created generated products from original files
- update to README to explain all of this
- clean up of old unused files

### CODE REVIEW

**Comment by ehsteve** [comment]
ping @Cadair

File: convert.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
```
Comment (dpshelio) [inline convert.sh]: I'm not sure `sh` should be the appropriate extension for this file, why not call it `.py`?

File: convert.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+for f in files:
+    os.remove(f)
+
+
+source_files = glob.glob('*.svg')
+print(source_files)
+# Convert form Inkscape SVG to PNG
+#inkscape sunpy_logo.svg --export-png=generated/sunpy_logo.png
+for f in source_files:
+    subprocess.call(["inkscape", f, "--export-png=generated/" + os.path.splitext(f)[0] + ".png"])
```
Comment (dpshelio) [inline convert.sh]: The readme doesn't mention `inkscape` is needed

**Review by dpshelio — COMMENTED** [review]
Should the font be added to the repository as a submodule? That way would be easy to check how it works if updated in upstream.
Also, should the generated ones be build in travis and push to a `generated` branch? So it can be used externally, but the code is cleaner - and we are sure when someone does a PR the generated output is the one said and not the one pushed.

**Comment by ehsteve** [comment]
@dpshelio I think it would great to have Travis build the generated files automatically though I'm not really sure how to do that. Have you done that before?

**Comment by dpshelio** [comment]
I've done stuff like generate things and then push it to a branch within the repository.

**Comment by ehsteve** [comment]
Can you show me an example? How can travis push to a branch? How do the permissions work?

**Comment by dpshelio** [comment]
@ehsteve - I've done it in the blog fetcher for GSOC: https://github.com/OpenAstronomy/Universe_OA there travis has a encrypted file that then converts to something acceptable from github.
More info in [Travis - Encrypting files](https://docs.travis-ci.com/user/encrypting-files/) - but I think we could do it later on, up to you :)

File: convert.sh (hunk 2 above)
Comment (ehsteve) [reply convert.sh]: fixed it.

File: convert.sh (hunk 1 above)
Comment (ehsteve) [reply convert.sh]: well its a shell script...not sure what the convention is here.

**Review by Cadair — APPROVED** [review]

**Comment by ehsteve** [comment]
@dpshelio need one more approval!

**Review by dpshelio — APPROVED** [review]

**Pull request merged**